.github/workflows/build.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   build:
+    name: Build
```

Comment:
Does GitHub stop running the action if it doesn't see the name or the name is not capitalized?

I do not see any value added by this PR. And it wastes the CI resource and the review time.

Reply:
Having a combination of lowercase and uppercase does not look professional.

<img width="985" alt="Screen Shot 2021-02-22 at 11 58 51 am" src="https://user-images.githubusercontent.com/418747/108647882-74374e80-7505-11eb-9c32-1137967dfcfb.png">